.devcontainer/devcontainer.json

```
@@ -0,0 +1,50 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/cpp
+{
+	"name": "C++",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Debian / Ubuntu OS version: debian-11, debian-10, ubuntu-22.04, ubuntu-20.04, ubuntu-18.04
+		// Use Debian 11, Ubuntu 18.04 or Ubuntu 22.04 on local arm64/Apple Silicon
+		"args": {
+			"VARIANT": "ubuntu-22.04",
+			"PROXY": "${localEnv:DEVCONTAINER_PROXY}",
+			"PROXY_PORT": "${localEnv:DEVCONTAINER_PROXY_PORT}",
+			"PROXY_HOST": "${localEnv:DEVCONTAINER_PROXY_HOST}"
+		}
+	},
+	"runArgs": [
+		"--cap-add=SYS_PTRACE",
+		"--security-opt",
+		"seccomp=unconfined"
+	],
+	"features": {
+		"docker-in-docker": {
+			"version": "latest",
+			"moby": true
+		}
```

This way you do not need to install python via custom scripts:

```suggestion
		},
		"ghcr.io/devcontainers/features/python:1": "os-provided"
```

---

Done